tasks/terraform_apply.json

```diff
@@ -0,0 +1,27 @@
+{
+  "description": "Apply an HCL manifest",
```

Comment:
I don't feel strongly about this, but I like the plan naming better than the task naming. `terraform::apply` was what I initially thought the task was called (without looking before running it), and I think since it's in the TF module it won't be confused with puppet apply. Either way it seems like the plans and task should have the same name scheme?

Reply:
Agreed, updated task names to exclude `terraform` prefix